include/swift/AST/ConstTypeInfo.h

```diff
@@ -194,7 +195,7 @@ class RuntimeValue : public CompileTimeValue {
 };
 
 struct CustomAttrValue {
-  swift::Type Type;
+  swift::CustomAttr *Attr;
```

Comment:
This makes me think that it is useful to gather all custom attributes here. It's fine to break them up into property wrappers separately, as we do here. But then in the resulting output we would have a separate category for `PropertyWrappers` and `OtherAttributes`. But that doesn't need to be a part of this change.